lib/mdx.ml

```diff
@@ -73,31 +78,103 @@ let parse l =
       | `Block b   -> Block b
     ) l
 
+let parse_mli file_contents =
```

Comment:
This could probably go `Mli_parser`.

It would be nice to have some unit tests for this btw, maybe for an intermediate step since `Block.t` are a bit heavyweight.

Reply:
:+1: Moved it out and wrote some unit tests